src/ios/BranchSDK.m

```diff
@@ -323,20 +311,17 @@ - (void)createBranchUniversalObject:(CDVInvokedUrlCommand*)command
 
   for (id key in properties) {
     if ([key isEqualToString:@"contentMetadata"]){
-      NSDictionary *metadata = (NSDictionary *)[properties valueForKey:key];
```

Comment:
I'd like to see this part tested in a either the testbed or your own test app. The slight change in logic looks safe, but really want to be sure the command actually has the right data type in it.

Reply:
Confirmed that setting contentMetadata still works and appears properly in requests.